### ISSUE TITLE
Doesn't work with Django 3.0

### ISSUE THREAD

**Issue by maxmalysh** [skorokithakis/django-loginas#87]
Tried the latest django-loginas (master branch of this repo) and got this error:

```
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/mmalysh/Development/mining-website/kryptex/config/urls.py", line 57, in <module>
    url(r'^adminium/', include('loginas.urls')),
  File "/Users/mmalysh/Development/mining-website/venv/lib/python3.6/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/Users/mmalysh/.pyenv/versions/3.6.9/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/mmalysh/Development/mining-website/venv/lib/python3.6/site-packages/loginas/urls.py", line 2, in <module>
    from loginas.views import user_login, user_logout
  File "/Users/mmalysh/Development/mining-website/venv/lib/python3.6/site-packages/loginas/views.py", line 5, in <module>
    from django.utils import six
ImportError: cannot import name 'six'
```

**Comment by skorokithakis** [comment]
I guess it's time to leave Python 2 support (and six) behind.

**Comment by skorokithakis** [comment]
Can you try the latest master and let me know if it works?

**Comment by maxmalysh** [comment]
Yeah, it works 👍

**Comment by skorokithakis** [comment]
Great, thank you. I will remove py2 support and release a new version soon.

**Comment by maxmalysh** [comment]
Thank you! 🙌

**Comment by treyrich** [comment]
Is there an ETA on the new version?

**Comment by skorokithakis** [comment]
Released 0.3.8!
